sim/rogue/subtlety/subtlety.go

```diff
@@ -3,9 +3,15 @@ package subtlety
 import (
 	"github.com/wowsims/cata/sim/core"
 	"github.com/wowsims/cata/sim/core/proto"
+	"github.com/wowsims/cata/sim/core/stats"
 	"github.com/wowsims/cata/sim/rogue"
 )
 
+const masteryDamagePerPoint = .01
+const masteryBaseEffect = 0.2
+
+var hasT6 bool = false
```

Comment:
global variable will break in raid sim with more then 1 rogue

Reply:
Also T6? I think we should remove anything that is not T10. Honestly even T10 is probably way behind the new 85 pre-raid gear so is it even needed?

Reply:
Definitely likely they can just be removed.

For Sub rogue specifically there's some tech around pre-casting a T6 buffed Slice and Dice due to HaT, but that can just be an Input option.